core/src/main/java/org/springframework/ldap/core/support/AbstractContextSource.java

```diff
@@ -119,10 +117,16 @@
 
 	public static final String SUN_LDAP_POOLING_FLAG = "com.sun.jndi.ldap.connect.pool";
 
-	private static final String JDK_142 = "1.4.2";
```

Comment:
Thanks for the cleanup -- will you please place this in a separate commit since it's not directly related to the change?

Reply:
Done